portworx-resize-pvc/step5.md

```diff
@@ -0,0 +1 @@
+##### Q5: Create a group snapshot called 'group-nginx-snap' for the PVC's of the cassandra StatefulSet. Make sure that the snapshot can be restored to a new namespace called 'nginx-clone'.
```

Comment:
This is confusing? a group snapshot of nginx for cassandra ss? Is this just a copy paste issue?

Reply:
This step is not used for this scenario (not used in the index.json ) This was copied from another scenario as a template. 

I wil lremove it.